flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -1265,11 +1266,9 @@ private ApplicationReport startAppMaster(
                     if (appState != lastAppState) {
                         LOG.info("Deploying cluster, current state " + appState);
                     }
-                    if (System.currentTimeMillis() - lastLogTime > 60000) {
-                        lastLogTime = System.currentTimeMillis();
+                    if (System.currentTimeMillis() - startTime > 60000) {
                         LOG.info(
-                                "Deployment took more than {} seconds. Please check if the requested resources are available in the YARN cluster",
-                                (lastLogTime - startTime) / 1000);
+                                "Deployment took more than 60 seconds. Please check if the requested resources are available in the YARN cluster");
```

Comment:
can i ask why do we need change this log report?

Reply:
I guess there is a `while(true)`, so the 60s can work well. And please @slfan1989 help double check.

Anyway, I think it was my mistake, this PR should only improve the memory unit and shouldn't include other changes.

Reply:
@1996fanrui Thanks for your attention. 

This will print log in every loop(250 milliseconds) if JobManager not started in 60 seconds. 
We reduced the frequency of printing logs in https://issues.apache.org/jira/browse/FLINK-30517

If these changes are not expected, Should I submit a pr to fix it?

Reply:
Sorry. 

From #21569 , I think this change is caused by rebase master. You can fix it, and please cc me after the PR is ready. And sorry again for the mistake.

Reply:
@1996fanrui @huwh Sorry, I didn't see the feedback question. When I rebase the master code later, I will be more careful.

Reply:
@slfan1989 Never mind.